validator/accounts/accounts_helper.go

```diff
@@ -202,21 +202,18 @@ func FilterExitAccountsFromUserInput(
 		return rawPubKeys, formattedPubKeys, nil
 	}
 
-	promptHeader := au.Red("===============IMPORTANT===============")
-	promptDescription := "Please navigate to the following website and make sure you understand the current implications " +
-		"of a voluntary exit before making the final decision:"
-	promptURL := au.Blue("https://docs.prylabs.network/docs/wallet/exiting-a-validator")
-	promptQuestion := "If you still want to continue with the voluntary exit, please input a phrase found at the above URL"
-	promptText := fmt.Sprintf("%s\n%s\n%s\n%s", promptHeader, promptDescription, promptURL, promptQuestion)
+	promptHeader := au.Red("===============CONFIRMATION NEEDED===============")
+	promptQuestion := "continue with the voluntary exit? (Y/n)"
+	promptText := fmt.Sprintf("%s\n%s", promptHeader, promptQuestion)
 	resp, err := prompt.ValidatePrompt(r, promptText, func(input string) error {
-		return prompt.ValidatePhrase(input, ExitPassphrase)
+		return prompt.ValidatePhrase(input, "Y", "Yes", "N", "No")
```

Comment:
Couldn't you just use `ValidateYesOrNo` and extend it to also allow for `yes` and `no`

Reply:
yeah i was thinking maybe it'd be used again, this lets you put in any number of valid phrases for the prompt, it only checks if it's nil or an error.

Reply:
We could always just leave it unused in case we ever need it again

Reply:
oops i missread this the first time let me adjust this then